### ISSUE TITLE
AutoExtensionImport undefined issue

### ISSUE THREAD

**Issue by andrewmanullang** [ampproject/amp-toolbox#1285]
Hey,

Is there any issue with `AutoExtensionImport`, since all injected script version are getting `undefined`, making the pages invalid

<img width="629" alt="Screen Shot 2022-01-20 at 15 09 48" src="https://user-images.githubusercontent.com/502997/150354096-54438e51-8655-4d2c-b6e1-f7301413e02b.png">

I'm using toolbox-optimizer 2.8.1

Any info regarding this? Thanks

**Comment by sebastianbenz** [comment]
Yes, this was a regression in the last AMP release, which got [fixed](https://github.com/ampproject/amphtml/issues/37415). Does the problem still persist?

**Comment by andrewmanullang** [comment]
Yes, the issue was still persist yesterday, so as a quick solution was to explicitly define all imports and it fixed the issue.

is there anything I should do to get the fix (e.g. update a version) or it's automatically patched?
